code/game/objects/items/weaponry.dm

```diff
@@ -315,7 +315,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	force = 2
 	throwforce = 20 //20 + 2 (WEIGHT_CLASS_SMALL) * 4 (EMBEDDED_IMPACT_PAIN_MULTIPLIER) = 28 damage on hit due to guaranteed embedding
 	throw_speed = 4
-	embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 100, "embedded_fall_chance" = 0)
+	embedding = list("embedded_pain_multiplier" = 1, "embed_chance" = 100, "embedded_fall_chance" = 0)
```

Comment:
how is this related to pethats?

Reply:
its... not where did this come from???

Reply:
I don't know, you commited this

Reply:
oh right it's a future commit. must have acidentally ploped it to this branch and it added it. compleatly forgot about it